### ISSUE TITLE
How to change the rec_ids for test_set?

### ISSUE THREAD

**Issue by KinWaiCheuk** [jthickstun/pytorch_musicnet#5]
In you paper, you only use 3 songs (2303, 2382, 1819) for testing.
But in your pytorch dataloader, you have 10 songs (2628, 2382, 2191, 2416, 2298, 1819, 2556, 2106, 1759, 2303).

How do I change which songs to test? I tried moving all the unwanted songs from `test_data `and `test_labels` to `train_data` and `train_labels`. But it still doesn't work.

I think I also need to modify the `test_tree.pckl`? How do I properly modify this file? I haven't used .pckl

**Comment by jthickstun** [comment]
We reported on this larger test set of 10 songs in our ICASSP paper:

[https://arxiv.org/abs/1711.04845](https://arxiv.org/abs/1711.04845)

I'd encourage you to work with the larger test set. If you must use the smaller test set, you can do the following:

(1) Move the relevant .wav and .npy files from the musicnet/test_data directory to the musicnet/train_data directory and from musicnet/test_labels directory to musicnet/train_labels directory.

(2) Check out the latest version of this repo, and load the dataset with the newly added "refresh_cache" flag set:

`    train_set = musicnet.MusicNet(root=root, train=True, download=True, refresh_cache=True, window=window)`

You only need to run with the refresh_cache=True flag once; it rebuilds the cached label pckl files that will pick up the changes you made in step 1.
